### PR TITLE
Update to shared values

### PR DESCRIPTION
**Pull request by phoenixbox**
I found the long sentences hard to read

I have multi lined them and added some visual differentiation which others might find useful

### CODE REVIEW

**Comment by patrickkettner** [comment]
lookin great :+1:

I think there is a couple small grammar nits, otherwise im all for it

File: README.md [hunk 1]
```diff
@@ -273,9 +273,45 @@ If you have values that you would like to share between various configuration ob
...
+* Result will be:
+
+```json
+{ 
+	"logLevel": "debug",
+	"logLocation": "/logs"
+}
+```
+
+If same key occurs in `$base` and the `filtered value` then value in `$base` will be overridden.
```
Comment (patrickkettner) [inline README.md]: I think this should be "If _the_ same", right?

**Comment by patrickkettner** [comment]
just the one thing - could you update and rebase down to a single commit?

cheers!

File: README.md (hunk 1 above)
Comment (phoenixbox) [inline README.md]: Sounds good :+1:

**Pull request merged**